nucypher/cli/options.py

```diff
@@ -182,22 +184,18 @@ def wrapper(**kwargs):
     return _decorator
 
 
-def process_middleware(mock_networking):
+def process_middleware(mock_networking) -> tuple:
+    """Must not raise"""
```

Comment:
Maybe a comment instead of docstring?

```suggestion
    #################
    # MUST NOT RAISE!
    #################
```

Reply:
:heavy_check_mark: